chapter_09_docker.asciidoc

```diff
@@ -927,7 +928,7 @@ WORKDIR /src
 CMD python manage.py runserver 8888
 ----
 ====
-
+//RITA: Newbie question. What does Ctrl+C do again? Add a couple of words to give us some quick context. "Use Ctrl+C to kill the current..."
```

Comment:
ctrl+c is so common in techie circles i find it hard to imagine anyone that hasn't come across it yet!  maybe if this was a pure beginners book...